09_json/mohitnag/pkg/puppy/types.go

```diff
@@ -0,0 +1,12 @@
+package puppy
+
+// Puppy defines the data structure corresponding to a Puppy
+type Puppy struct {
+	ID     uint32 `json:"puppyid"`
+	Breed  string `json:"breed"`
+	Colour string `json:"colour"`
+	Value  string `json:"value"`
+}
+
+// Puppies is an array of Puppy
+type Puppies []Puppy
```

Comment:
This type is used only in `main.go` and only in one place. This does not warrant creating a type for this. Just use `[]Puppy{}` in `main.go`, which is clearer there what is going on.

Reply:
Done